weft/handlers.go

```diff
@@ -287,6 +291,23 @@ func setBestPracticeHeaders(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Referrer-Policy", "no-referrer")
 }
 
+func logRequest(r *http.Request) {
+	logger.Printf("%s - %s - %s\n", r.RemoteAddr, r.Method, r.RequestURI)
+	if logPostBody {
```

Comment:
Could we have an `if` for normal request logging as well? Otherwise if you want to just turn on error logging you'll get all the request logs.